buddyboss/buddyboss-platform
# PROD-937 #1172 - Fix Bug - conflict with 'Invite Anyone' plugin

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1172.

### How to test the changes in this Pull Request:
1. Install plugin "Invite Anyone"
2. Enable Learndash/BuddyBoss group syncing
3. Try to create a group, see above errors

### Proof Screenshots or Video

https://www.loom.com/share/784556d1385b4e739b134a7343588498

### Other information:




### Changelog entry

>Fixed - conflict with 'Invite Anyone' plugin
